lib/matplotlib/backend_bases.py

```diff
@@ -3047,15 +3048,21 @@ def release_zoom(self, event):
             a._set_view_from_bbox((lastx, lasty, x, y), direction,
                                   event.key, twinx, twiny)
 
-        self.draw()
+        self._draw()
         self._xypress = None
         self._button_pressed = None
 
         self.push_current()
         self.release(event)
 
+    @cbook.deprecated("3.3", alternative="toolbar.figure.canvas.draw_idle()")
     def draw(self):
         """Redraw the canvases, update the locators."""
+        self._draw()
+
+    # Can be removed once Locator.refresh() is removed, and replaced by an
+    # inline call to self.figure.canvas.draw_idle().
```

Comment:
Here and above, wouldn't it be `self.canvas.draw_idle()`?

Reply:
yes, thanks for noticing.